Mlem/App/Views/Root/Tabs/Feeds/FeedsView.swift

```diff
@@ -59,35 +59,65 @@ struct MinimalPostFeedView: View {
     var body: some View {
         content
             .background(tilePosts ? palette.groupedBackground : palette.background)
-            .navigationTitle("Feeds")
             .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarEllipsisMenu {
+                    MenuButton(action: BasicAction(
+                        isOn: showRead,
+                        label: showRead ? "Hide Read" : "Show Read",
+                        color: palette.primary,
+                        icon: Icons.read
+                    ) {
+                        showRead = !showRead
+                    })
+                }
+            }
             .onChange(of: tilePosts, initial: true) { _, newValue in
                 if newValue {
-                    columns = [GridItem(.flexible(), spacing: 0), GridItem(.flexible(), spacing: 0)]
+                    // leading/trailing alignment makes them want to stick to each other, allowing the AppConstants.halfSpacing padding applied below
+                    // to push them apart by a sum of AppConstants.standardSpacing
+                    columns = [
+                        GridItem(.flexible(), spacing: 0, alignment: .trailing),
+                        GridItem(.flexible(), spacing: 0, alignment: .leading)
+                    ]
                 } else {
                     columns = [GridItem(.flexible())]
                 }
             }
-            .task {
-                if postTracker.items.isEmpty, postTracker.loadingState == .idle {
-                    print("Loading initial PostTracker page...")
-                    do {
-                        try await postTracker.loadMoreItems()
-                    } catch {
-                        handleError(error)
+            .task(id: showRead) {
+                scrollToTopTrigger.toggle()
```

Comment:
Navigating back from an expanded post or switching tab causes the page to scroll to top. Maybe using `.onChange` here instead would fix this?